maestrowf/maestro.py

```diff
@@ -61,16 +62,89 @@
 
 def status_study(args):
     """Check and print the status of an executing study."""
-    status = Conductor.get_status(args.directory)
-
-    if status:
-        print(tabulate.tabulate(status, headers="keys"))
-        return 0
+    
+    args_dir = args.directory
+
+    #Step 1: Find if the directory string contains an asterisk
+    contains_asterisk = False
+    ts_first = ""
+    ts_last = 0
+
+    if "*" in args_dir:
```

Comment:
Looking over this -- I believe if `glob.glob` is passed a fixed path, the `glob` will return a single entry in the generator. That change will get rid of the need for the conditional checking for the dashes and other characters because it will allow `glob` to do the heavy lifting.